modules/veteran/app/workers/veteran/base_reloader.rb

```diff
@@ -23,16 +23,10 @@ def find_or_initialize(hash)
     def fetch_data(action)
       page = Faraday.new(url: BASE_URL).post(action, id: 'frmExcelList', name: 'frmExcelList').body
       doc = Nokogiri::HTML(page)
-      content = CSV.generate(headers: true) do |csv|
-        doc.xpath('//table/tr').each do |row|
-          tarray = []
-          row.xpath('td').each do |cell|
-            tarray << cell.text.scrub
-          end
-          csv << tarray
-        end
+      headers = doc.xpath('//table/tr').first.children.children.map {|header| header.text }
+      doc.xpath('//table/tr').map do |row|
+        Hash[headers.zip(row.children.children.map {|cell| cell.text})]
```

Comment:
you dropped .scrub ... was that intentional?

Reply:
Yes, scrub was removing the \n and \r left overs that were left over in the string, actually going through the children.children removes all of that and leaves you only with the cell text itself.

Reply:
Scrub wasn't removing the newlines, it was removing invalid byte sequences. I believe it was a badly encoded apostrophe. We probably need to add it back